pkg/grpc/grpc.go

```diff
@@ -87,6 +93,10 @@ func NewGRPCServersFromConfigurationAndServe(configurations []*configuration.GRP
 		grpc_prometheus.EnableHandlingTimeHistogram(grpc_prometheus.WithHistogramBuckets(prometheus.DefBuckets))
 		grpc_prometheus.Register(s)
 
+		if len(configuration.ListenAddresses)+len(configuration.ListenPaths) == 0 {
+			return errors.New("GRPC server configured without any listen addresses or paths")
```

Comment:
I think this still needs to be changed to `status.Error()` and we're good to go!